metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -313,20 +314,23 @@ void main() {
     test(
         ".filterByProjectName() filters list of the project metrics according to the given value",
         () async {
+      final metricsTileViewModel =
+          projectMetricsNotifier.projectsMetricsTileViewModels.last;
+
       final expectedProjectMetrics = [
```

Comment:
Remove the unnecessary empty line.

```suggestion
      final metricsTileViewModel =
          projectMetricsNotifier.projectsMetricsTileViewModels.last;
      final expectedProjectMetrics = [
```